src/InterruptAccelStepper.h

```diff
@@ -64,7 +67,15 @@ template <typename STEPPER> class InterruptAccelStepper
     {
         LOG(DEBUG_STEPPERS, "[IAS-%d] setSpeed(%f)", STEPPER::TIMER_ID, speed);
         _speed = speed;
-        STEPPER::moveTo(_speed, INT32_MAX);
+
+        if (abs(speed) < MIN_STEPS_PER_SEC)
```

Comment:
Why do you use Arduino's `abs()` but also you define `ABS()` in this file? Just kind of generally confused what's going on with the usage

Reply:
to be honest atm i forgot why i defined my custom one. i think back then the arduino one could not be found for some stupid reason ... and this time i just used the arduino abs as usual

Reply:
i removed the custom one

Reply:
Shouldn't it be fabs()?

Reply:
well ... in this case then even `fabsf` (for float instead of double)

Reply:
done